docs/mint.json

```diff
@@ -69,7 +69,7 @@
     },
     {
       "name": "All Tools",
-      "url": "https://composio.dev/tools"
+      "url": "https://app.composio.dev/apps"
```

Comment:
Consider adding a comment block at the top of the configuration file documenting the URL structure change and the rationale. This will help future maintainers understand the URL routing strategy:

```json
/*
 * URL Configuration (Updated)
 * Base URL: app.composio.dev
 * Tools/Apps path: /apps
 * Previous path (/tools) is now redirected to /apps
 * All tool-related content is consolidated under app.composio.dev
 */
```